shared/chat/inbox/row/big-team-rows.js

```diff
@@ -22,7 +22,7 @@ class BigTeamHeaderRow extends PureComponent<TeamProps> {
           onClick={this.props.onShowMenu}
           style={{
             fontSize: isMobile ? 20 : 16,
-            padding: isMobile ? 2 : 0,
+            padding: isMobile ? 8 : 4,
```

Comment:
I think this will push the icon to the left though. Could we do all sides except the right side?

Reply:
👍